src/tof_camera_node.cpp

```diff
@@ -202,7 +216,18 @@ class TofNode : public rclcpp::Node
   {
     if (streamOnFlag) {
       globalTimeStamp = rclcpp::Clock{RCL_ROS_TIME}.now();
-      getNewFrame(camera, frame);
+
+      aditof::Frame * tmp1 = m_safeDataAccess.getNextElement();
+
+      auto start = high_resolution_clock::now();
+
+      m_safeDataAccess.setReadytoStart();
+      getNewFrame(camera, tmp1);
+      m_safeDataAccess.addElement(tmp1);
+
+      auto stop = high_resolution_clock::now();
+      auto duration = duration_cast<milliseconds>(stop - start);
+      LOG(INFO) << "getNewFrame duration ms=" << duration.count();
```

Comment:
Since the fps can reach values over 10, would it be advisable to print the value that many times?